Yoast/javascript
# Add missing specs for Hebrew, Hungarian, and Swedish morphology/readability files

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Add missing specs for Hebrew, Hungarian, and Swedish morphology/readability files.
* [Yoast SEO Premium] Improves Hungarian word form recognition of certain words with the letter combinations `cs`, `dz`, `dzs`, `gy`, `ly`, `ny`, `ty`, and `zs`, for example `balszerencséjük` and `balszerencséjére`.

## Relevant technical choices:

* This PR should be reviewed and merged together with this premium configuration [PR](https://github.com/Yoast/YoastSEO.js-premium-configuration/pull/223) because of changes to the Hungarian stemmer that also affected the morphology data.

## Test instructions

This PR can be tested by following these steps:

* In **Premium**, set your site to Hungarian
* Open a post and set your keyphrase to `balszerencséjük`.
* Paste `balszerencséjére` somewhere in the text.
* Make sure that the keyphrase density assessment shows that the keyphrase was found 1 time.
* Swap the keyphrase and the word in the text, and make sure the assessment result remains the same.

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes https://yoast.atlassian.net/browse/LINGO-656
